content/use-cases/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: >-
+  Use cases of Rhize
+description: >-
+  Handle manufacturing events, access the knowledge graph of the operation, build custom MOM applications. 
+weight: 10
+menu:
+  main:
+    identifier: use-case-overview
+    parent: use-cases
+    name: Overview
+---
+
+Rhize's flexible, event-centric architecture serves many functions.
+While Rhize has components that can replace an {{< abbr "MES" >}}, historian, andon, and real-time monitoring solutions,
+it can complement them just as well.
+You can map data from an MES or {{< abbr "ERP" >}} into the database, creating a coherent data model to unite your operations IT.
+
+Besides better performance and flexibility, Rhize has far tighter integration of plant and system data.
+Yet, its data model is generic enough to conform to may use cases, chiefly:
+- **A manufacturing knowledge graph**. Query the entire context and history of the operation.
+- **An event handler**. Receive manufacturing message streams and react to them.
+- **Headless MES or MOM**. Use the API to build custom applications for a variety of MES and MOM activities.
+
+Of course, each of these use cases has many uses cases.
+These use cases of Rhize are already implemented in discrete, continuous, and batch manufacturing operations.
+
+## Manufacturing knowledge graph
+
+All data that Rhize collects, whether from sensors or an ERP, is contextual and interconnected. Rather than a relational database, Rhize uses a graph database, where any node can link to any other, and users can query any data combination, without requiring complex joins.
+
+The graph database unlocks new possibilities for manufacturing analysis and data science.
+For example:
+- Run queries to find anomalies in an operation---which may trace to a specific site, segment, equipment, material lot, personnel, and so on.
+- Discover places to optimize the system, whether they are bottlenecks to remove or highly productive areas to replicate
+- Train deep-learning models to detect conditions that lead to batch failures.
+- Use the historical record as a model to run simulations of new events
+
+Guide: [Use the knowledge graph]({{< relref "../how-to/gql" >}})
+
+## Headless MES or MOM
+
+Rhize serves as a backend to create custom applications to replace traditional MES or MOM systems.
```

Comment:
```suggestion
Rhize serves as a backend to create custom applications to replace traditional MES or MOM systems.
Rather than force its opinion of what an MES interface should look like, Rhize provides the only data model, API, and BPMN engine.
Your frontend teams can then use the tools of their choice to make the MES designed for your use case, with all the backend work delegated to Rhize's normal operation features.

```